### PR TITLE
Moar `<FormattedDateString>`

### PR DESCRIPTION
**Pull request by mateoclarke**
## Associated issues

Closes https://github.com/cityofaustin/atd-data-tech/issues/19044

## Testing

**URL to test:** 

_Since this is a PR off a branch from main ([chia's PR](https://github.com/cityofaustin/atd-moped/pull/1438)), we'll have to test this locally until that upstream PR branch is merged to `main`_

**Steps to test:**

1. Check the **Modified** column and the date format for each row on the project list
  - https://localhost:3000/moped/projects
2. Check the **Date uploaded** column in the project files tab table for the expected date formatting
  - https://localhost:3000/moped/projects/227?tab=files
3. Check the 3rd column in the Project Activity log for the expected date formatting
  - ex: https://localhost:3000/moped/projects/227?tab=activity_log

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Product manager checked [DB view dependencies](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)
- [x] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectsListView/helpers.js [hunk 1]
```diff
@@ -246,8 +246,14 @@ export const useColumns = ({ hiddenColumns }) => {
         headerName: "Modified",
         description: "Date record was last modified",
         field: "updated_at",
-        valueFormatter: (value) => formatTimeStampTZType(value),
         width: COLUMN_WIDTHS.small,
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectsListView/helpers.js:249]: this wasn't in the issue, but i'm now noticing that we could update this to `COLUMN_WIDTHS.medium` to keep the secondary part of the timestamp from wrapping in cell when its default width loads. Could you make that change? Thanks!

File: moped-editor/src/views/projects/projectsListView/helpers.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { useMemo } from "react";
 import { NavLink as RouterLink, useLocation } from "react-router-dom";
 import Link from "@mui/material/Link";
 import parse from "html-react-parser";
-import { formatDateType, formatTimeStampTZType } from "src/utils/dateAndTime";
+import { formatDateType, FormattedDateString, formatTimeStampTZType } from "src/utils/dateAndTime";
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectsListView/helpers.js:5]: I'm seeing a compile warning that `formatTimeStampTZType` isn't being used. could you remove? 🙏

**Review by mddilley — CHANGES_REQUESTED** [review]
I requested two tiny changes - this looks amazing! Thank you for spreading this component to more places! 🎉

**Comment by chiaberry** [comment]
I merged the upstream branch and switched the base branch to main, hoping it would trigger a netlify deploy but alas no

**Review by mddilley — APPROVED** [review]
Love seeing this component in more places - thanks! ⏳ 🙌 🚢

**Pull request merged**